### ISSUE TITLE
Problem installing on minty ubuntu 10.10

### ISSUE THREAD

**Issue by OriPekelman** [af83/auth_server#2]
Steps to repeat:
`
sudo apt-get install git gettext build-essentials
git clone git://github.com/creationix/nvm.git ~/.nvm
 . ~/.nvm/nvm.sh
nvm sync
nvm install stable
. ~/.nvm/nvm.sh
cd ~
git clone git://github.com/AF83/auth_server.git
cd auth_server
make install
make update_js_templates
`
Here First problem:
`node src/scripts/load_data.js`

gave me:
`
node.js:134
        throw e; // process.nextTick error, or 'error' event on first tick
        ^
Error: Cannot find module 'mongodb'
    at Function._resolveFilename (module.js:322:11)
    at Function._load (module.js:267:25)
    at require (module.js:351:19)
    at Object.<anonymous> (/home/ori/auth_server/node_modules/mongodb-provider/lib/provider.js:4:13)
    at Object.<anonymous> (/home/ori/auth_server/node_modules/mongodb-provider/lib/provider.js:108:4)
    at Module._compile (module.js:407:26)
    at Object..js (module.js:413:10)
    at Module.load (module.js:339:31)
    at Function._load (module.js:298:12)
    at require (module.js:351:19)
`
`npm list` gave a list of broken dependencies:
`AuthServer@0.1.2 /home/ori/auth_server
├── base64@2.0.3 
├─┬ connect@1.4.1 
│ ├── UNMET DEPENDENCY mime >= 0.0.1
│ └── UNMET DEPENDENCY qs >= 0.0.6
├── connect-form@0.2.1 
├── connect-sts@0.4.0 
├── cookie-sessions@0.0.2 
├── eyes@0.1.6 
├── formidable@1.0.2 
├─┬ futures@2.0.3 
│ └── require-kiss@1.0.5 
├── glob@2.0.6 
├─┬ mail@0.2.2 
│ └── reparse@0.1.2 
├─┬ mongodb-provider@0.1.1  invalid
│ └── UNMET DEPENDENCY mongodb >=0.9.1
├── mustache@0.3.1-dev 
├─┬ oauth2-client@0.0.4  invalid
│ └── serializer@0.0.2 
├─┬ oauth2-server@0.0.1  invalid
│ └── serializer@0.0.2 
└── request@1.9.5 
`

`rm -rf node_modules/`
and 
`make install` 
again fixed this

but got with npm ls

`AuthServer@0.1.2 /home/ori/auth_server
├── base64@2.0.3 
├─┬ connect@1.4.1 
│ ├── mime@1.2.2 
│ └── qs@0.1.0 
├── connect-form@0.2.1 
├── connect-sts@0.4.0 
├── cookie-sessions@0.0.2 
├── eyes@0.1.6 
├── formidable@1.0.2 
├─┬ futures@2.0.3 
│ └── require-kiss@1.0.5 
├── glob@2.0.6 
├─┬ mail@0.2.2 
│ └── reparse@0.1.2 
├─┬ mongodb-provider@0.1.1  invalid
│ └── mongodb@0.9.4-5 
├── mustache@0.3.1-dev 
├─┬ oauth2-client@0.0.4  invalid
│ └── serializer@0.0.2 
├─┬ oauth2-server@0.0.1  invalid
│ └── serializer@0.0.2 
└── request@1.9.5`

Running
`node src/server.js`

gives us:

`node.js:134
        throw e; // process.nextTick error, or 'error' event on first tick
        ^
Error: TypeError: Cannot read property 'cursorId' of null
    at [object Object].nextObject (/home/ori/auth_server/node_modules/mongodb-provider/node_modules/mongodb/lib/mongodb/cursor.js:474:16)
    at Array.<anonymous> (/home/ori/auth_server/node_modules/mongodb-provider/node_modules/mongodb/lib/mongodb/cursor.js:171:12)
    at EventEmitter._tickCallback (node.js:126:26)
`

**Comment by francois2metz** [comment]
Ok I can reproduce the error.

Please always provide the log.

```
> mongodb@0.9.4-5 install /home/francois/dev/turbulences/auth_server/node_modules/mongodb-provider/node_modules/mongodb
> bash ./install.sh
./install.sh: line 2: [: too many arguments
```

**Comment by francois2metz** [comment]
Should be fixed by cfd9773fad933147a0a132797f41d42fac324203. I have released the version 0.1.3.
